src/prettyPrinter.ts

```diff
@@ -480,9 +480,55 @@ export const ppAstAsmFunctionDef: Printer<A.AstAsmFunctionDef> =
             c.row(
                 `asm${ppAsmShuffle(node.shuffle)} ${ppAstFunctionSignature(node)} `,
             ),
-            ppAsmInstructionsBlock(node.instructions)(c),
+            ppAsmExpressionsBlock(node.expressions)(c),
         ]);
 
+// NOTE: ok, I might need some help here...
+export const ppAstAsmExpressionList: Printer<A.AstAsmExpressionList> =
```

Comment:
`c.braced(node.exprs.map(ppAstAsmExpression))`

Reply:
Depending on the order PRs get merged, you might not have this problem here at all :)

Reply:
Thanks, but I've already removed the asm expression lists from the grammar, such that `{ ... }` are prohibited. At least until we may or may not make sense of them ourselves and without Fift's aid.